### ISSUE TITLE
reading MFCC features from CSV files instead of extracted them on the fly.  

### ISSUE THREAD

**Issue by kerolos** [keras-team/keras-io#1276]
In this Tutorial ([keras-io](https://github.com/keras-team/keras-io)/[examples](https://github.com/keras-team/keras-io/tree/master/examples)/[audio](https://github.com/keras-team/keras-io/tree/master/examples/audio)/ctc_asr.py) 
https://keras.io/examples/audio/ctc_asr/#:~:text=CTC%20is%20an%20algorithm%20used,create%20is%20similar%20to%20DeepSpeech2.

The features had been extracted from wav files path with is in dataframe :
def encode_single_sample(wav_file, label):
     file = tf.io.read_file(wavs_path + wav_file + ".wav")
     
Is there any way to read the features MFCC from CSV files in this function 
 feature_dim=41
 font_column_types= [float()]*(feature_dim)
 numeric_features = tf.io.decode_csv(feature_file,record_defaults=font_column_types)

**Comment by sampathweb** [comment]
Tagging the author of the tutorial - @rbouadjenek

**Comment by rbouadjenek** [comment]
@kerolos @sampathweb 

It it possible to do a two stage preprocessing. You just need to adapt the code 🙂🙂
